platform_channels/test/src/pet_list_screen_test.dart

```diff
@@ -22,12 +22,12 @@ void main() {
       }
     ];
 
-    PetListModel petListModel;
+    PetListModel? petListModel;
```

Comment:
Can we convert this from optional to `late`?

Reply:
Sure thing!

Reply:
Hey, @domesticmouse I have updated the PR  with the new changes but after converting the  ` PetListModel? petListModel`  to ` late` a couple of tests are getting failed would you please have a look into the issue.
```shell
══╡ EXCEPTION CAUGHT BY FLUTTER TEST FRAMEWORK ╞════════════════════════════════════════════════════
The following LateError was thrown running a test:
LateInitializationError: Local 'petListModel' has not been initialized.
```

Reply:
So the answer is no, it can't be converted to `late`. Revert the change =)

Reply:
done @domesticmouse!